.github/workflows/bump-otel-version.yml

```diff
@@ -0,0 +1,63 @@
+name: bump-otel-version
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '24 5 * * 1-5'
+
+permissions:
+  contents: read
+
+env:
+  JOB_URL: "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+
+jobs:
+  filter:
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    outputs:
+      matrix: ${{ steps.generator.outputs.matrix }}
+    permissions:
+      contents: read
+    steps:
+    - id: generator
+      uses: elastic/oblt-actions/elastic/active-branches@v1
+      with:
+        # Only run on `main` and `8.19` branches
+        exclude-branches: "7.17,8.17,8.18,9.0"
```

Comment:
Is the idea here to backport the PR from `main` to active `9.*` branches?

Reply:
Yes, I'm not a fan of this approach used in this PR: "get all the active branches, and exclude ... everything apart from two". I think it would be better to explicitly state the branches that we want this workflow to be run for (currently `main` and `8.19`). I used this solution for consistency with approach used in https://github.com/elastic/elastic-agent/pull/8174/files#diff-e2c8c610c077a247bb8aa660cbffc7a1e2bc16e5dc4e61069087cfe03e3293faR23.

I'll happily change using the `elastic/oblt-actions/elastic/active-branches` action to specifying a static matrix with `main` and `8.19`. Would that work?